runtime/vm/BytecodeInterpreter.hpp

```diff
@@ -5153,38 +5153,39 @@ class INTERPRETER_CLASS
 	VMINLINE VM_BytecodeAction
 	enterContinuationImpl(REGISTER_ARGS_LIST)
 	{
-		VM_BytecodeAction rc = EXECUTE_BYTECODE;
-		jboolean res = JNI_FALSE;
+		VM_BytecodeAction rc = GOTO_RUN_METHOD;
```

Comment:
I think this would look better if you init to execute bytecode and set the rc to run method at the same place where you set the method (gets rid of the else below).